File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/PlayerAnnotationsExpandedRenderer.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class PlayerAnnotationsExpandedRenderer{

	@SerializedName("featuredChannel")
	private FeaturedChannel featuredChannel;

	@SerializedName("annotationId")
	private String annotationId;

	@SerializedName("allowSwipeDismiss")
	private boolean allowSwipeDismiss;

	public void setFeaturedChannel(FeaturedChannel featuredChannel){
		this.featuredChannel = featuredChannel;
	}

	public FeaturedChannel getFeaturedChannel(){
		return featuredChannel;
	}

	public void setAnnotationId(String annotationId){
		this.annotationId = annotationId;
	}

	public String getAnnotationId(){
		return annotationId;
	}

	public void setAllowSwipeDismiss(boolean allowSwipeDismiss){
		this.allowSwipeDismiss = allowSwipeDismiss;
	}

	public boolean isAllowSwipeDismiss(){
		return allowSwipeDismiss;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/PlayerAnnotationsExpandedRenderer.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class PlayerAnnotationsExpandedRenderer : Parcelable, Serializable {
    @SerializedName("featuredChannel")
    var featuredChannel: FeaturedChannel? = null

    @SerializedName("annotationId")
    var annotationId: String? = null

    @SerializedName("allowSwipeDismiss")
    var isAllowSwipeDismiss = false
    override fun toString(): String {
        return "PlayerAnnotationsExpandedRenderer{" +
                "featuredChannel = '" + featuredChannel + '\'' +
                ",annotationId = '" + annotationId + '\'' +
                ",allowSwipeDismiss = '" + isAllowSwipeDismiss + '\'' +
                "}"
    }
}